applications/minotari_node/src/config.rs

```diff
@@ -143,6 +145,44 @@ impl Default for BaseNodeConfig {
             network: Network::default(),
             grpc_enabled: true,
             grpc_address: None,
+            grpc_server_config: BaseNodeGrpcServerConfig {
+                // These gRPC server methods will not share sensitive information, thus enabled by default
+                enable_list_headers: true,
+                enable_get_header_by_hash: true,
+                enable_get_blocks: true,
+                enable_get_block_timing: true,
+                enable_get_constants: true,
+                enable_get_block_size: true,
+                enable_get_block_fees: true,
+                enable_get_tokens_in_circulation: true,
+                enable_get_network_difficulty: true,
+                enable_get_new_block_template: true,
+                enable_get_new_block: true,
+                enable_get_new_block_blob: true,
+                enable_submit_block: true,
+                enable_submit_block_blob: true,
+                enable_submit_transaction: true,
+                enable_search_kernels: true,
+                enable_search_utxos: true,
+                enable_fetch_matching_utxos: true,
+                enable_get_peers: true,
+                enable_get_mempool_transactions: true,
+                enable_transaction_state: true,
+                enable_list_connected_peers: true,
+                enable_get_mempool_stats: true,
+                enable_get_active_validator_nodes: true,
+                enable_get_shard_key: true,
+                enable_get_template_registrations: true,
+                enable_get_side_chain_utxos: true,
+                // These gRPC server methods share sensitive information, thus disabled by default
+                enable_get_version: false,
```

Comment:
Why not just have a list of denied methods?

```rust
deny_methods: Vec<GrpcMethod>
```

and implement FromStr for GrpcMethod

Reply:
Thanks for the discussion, implemented